test/unittests/test_testing_tools.py

```diff
@@ -221,3 +228,109 @@ def test_compare_folders_structures(
     shutil.rmtree(tmp_path / "sub-02")
     with pytest.raises(ValueError, match="/sub-02/bar.tsv not found !"):
         compare_func(tmp_path, tmp_path / "hashes.pl")
+
+
+def build_bids_tsv(tmp_path: Path) -> Path:
+    bids_path = tmp_path / "BIDS"
+    bids_path.mkdir()
+    prpc = pd.DataFrame({"participant_id": ["sub-002", "sub-001"], "age": [20, 26]})
+    prpc.to_csv(bids_path / "participants.tsv", sep="\t", index=False)
+    sub_path = bids_path / "sub-001"
+    sub_path.mkdir()
+    sess = pd.DataFrame({"session_id": ["ses-M012", "ses-M006"], "age": [20, 20]})
+    sess.to_csv(sub_path / "sub-001_sessions.tsv", sep="\t", index=False)
+    ses_path = sub_path / "ses-M016"
+    ses_path.mkdir()
+    scans = pd.DataFrame(
+        {
+            "filename": ["pet/foo.json", "anat/foo.json"],
+            "acq_time": ["00:00:00", "00:00:00"],
+        }
+    )
+    scans.to_csv(ses_path / "sub-001_ses-M016_scans.tsv", sep="\t", index=False)
+    return bids_path
+
+
+def test_loaders(tmp_path):
```

Comment:
Could you split this test in three ? 
First, it'd be more clean and unitary and also, you'll see which loaders are broken the first time without having to re-run multiple times.